azure/components/userpool/setup.ftl

```diff
@@ -62,6 +65,51 @@
 
             [#local client = subResources["client"]]
 
+            [#list subSolution.Links?values as link]
+
+                [#local linkTarget = getLinkTarget(subOccurrence, link)]
+
+                [@debug message="Link Target" context=linkTarget enabled=false /]
+
+                [#if !linkTarget?has_content]
+                    [#continue]
+                [/#if]
+
+                [#local linkTargetCore = linkTarget.Core]
+                [#local linkTargetConfiguration = linkTarget.Configuration]
+                [#local linkTargetResources = linkTarget.State.Resources]
+                [#local linkTargetAttributes = linkTarget.State.Attributes]
+
+                [#switch linkTargetCore.Type]
+
+                    [#case LB_PORT_COMPONENT_TYPE]
```

Comment:
Do your load balancers currently support Authentiating on their own? if not I'd take this out

Reply:
Removed in #142